api/src/pcapi/routes/backoffice/templates/offerer/get/stats.html

```diff
@@ -1,107 +1,158 @@
 {% from "components/dynamic/modal.html" import build_dynamic_modal with context %}
+{% set search_params = {
+  "search-0-search_field": "OFFERER",
+  "search-0-operator": "IN",
+  "search-0-offerer": offerer.id,
+} %}
 <div class="row">
-  <div class="col-md-4">
+  <div class="col-md-6">
     <div class="card h-100 shadow">
-      <div class="card-body">
-        <h6 class="display-6 card-title">
-          {% if stats.total_revenue == stats.placeholder %}
-            <span data-bs-toggle="tooltip"
-                  data-bs-placement="top"
-                  data-bs-title="Il y a trop d'offres pour en afficher la valeur">N/A</span>
-          {% else %}
-            {{ stats.total_revenue | format_amount(target=offerer) }}
-          {% endif %}
-        </h6>
-        <div class="d-flex flex-row">
-          <div class="card-text fw-light small align-self-center">de CA</div>
-          <div class="p-auto">
-            {% if stats.total_revenue != stats.placeholder and stats.total_revenue > 0 %}
-              <button class="btn link-primary"
-                      data-bs-toggle="modal"
-                      data-bs-target="#display-revenue-details"
-                      type="button">plus de détails…</button>
-              {{ build_dynamic_modal(url_for("backoffice_web.offerer.get_revenue_details", offerer_id=offerer.id) , "display-revenue-details") }}
-            {% endif %}
+      <div class="card-body row">
+        <div>
+          <h3>Offres</h3>
+          <div class="d-flex justify-content-between">
+            <div>
+              <h1 class="fw-bold text-black">
+                {% if stats.active_individual_offers is not none  and stats.inactive_individual_offers is not none %}
+                  {{ stats.active_individual_offers + stats.inactive_individual_offers }}
+                {% else %}
+                  N/A
+                {% endif %}
+              </h1>
+              <div>
+                {% if has_permission("READ_OFFERS") %}
+                  <a class="link-primary"
+                     href="{{ url_for('backoffice_web.offer.list_offers', **search_params) }}">Individuelles</a>
+                {% else %}
+                  Individuelles
```

Comment:
Conformément à la maquette : 
```suggestion
                     href="{{ url_for('backoffice_web.offer.list_offers', **search_params) }}">individuelles</a>
                {% else %}
                  individuelles
```